modules/monitoring-configuring-shiftstack-scraping.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/shiftstack-prometheus-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="monitoring-configuring-shiftstack-scraping_{context}"]
+= Collecting cluster metrics from the federation endpoint
+
+// TODO: Is this RHOSO only?
+
+You can open the federation endpoint of a cluster to make metrics available to a Red Hat OpenStack Services on OpenShift (RHOSO) cluster to practice pull-based monitoring.
```

Comment:
Do we need to make it clear this is an alternative to the above method of sending metrics to an external Prometheus server?

Reply:
Can't hurt.